content-src/styles/_mixins.scss

```diff
@@ -15,3 +15,9 @@
   max-height: 1em * $line-count * $line-height / $font-size;
   overflow: hidden;
 }
+
+@mixin dark-theme-only {
+  [lwt-newtab-brighttext]:not(.force-light-theme) & {
```

Comment:
Can you explain this bit? I get what it does based on the name of the mixin, I just don't really see how this selector does that.

Reply:
It's based on an attribute that gets added to the body tag for dark mode.